src/jsTest/kotlin/it/krzeminski/githubactionstyping/AjvSchemaValidationTest.kt

```diff
@@ -0,0 +1,100 @@
+package it.krzeminski.githubactionstyping
+
+import ajv.Ajv
+import ajv.ValidateFunction
+import io.kotest.assertions.withClue
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.Matcher
+import io.kotest.matchers.MatcherResult
+import io.kotest.matchers.booleans.shouldBeTrue
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.should
+import io.kotest.matchers.shouldNot
+import io.kotest.mpp.env
+import node.buffer.BufferEncoding.Companion.utf8
+import node.fs.exists
+import node.fs.existsSync
+import node.fs.opendir
+import node.fs.readFile
+import node.fs.readdirSync
+import node.fs.stat
+import node.path.path
+import ajv.Options as AjvOptions
+
+private val schemaFile = env("schemaFile")!!
+private val catalogDir = env("catalogDir")!!
+private val externalDir = env("externalDir")!!
+private val goodDir = env("goodDir")!!
+private val badDir = env("badDir")!!
+
+private lateinit var validate: ValidateFunction
+
+/**
+ * Validate good and bad typings against the Ajv schema validator.
+ */
+class AjvSchemaValidationTest : FunSpec({
```

Comment:
Do you also get this when running from IntelliJ?
<img width="1213" alt="Screenshot 2025-01-29 at 08 30 26" src="https://github.com/user-attachments/assets/155c9a60-87e4-4608-998b-6911ebbaa5b8" />

Reply:
Actually, when I try to run from IntelliJ I get "'Kotlin SDK' is misconfigured" and it does not run, so I only start the test through Gradle even within IDE.

Reply:
Ok, let's ignore this for now.